mmdet/models/detectors/centernet2.py

```diff
@@ -0,0 +1,41 @@
+from ..builder import DETECTORS
+from .two_stage import TwoStageDetector
+
+
+@DETECTORS.register_module()
+class CenterNet2(TwoStageDetector):
+    """Implementation of `Probabilistic two-stage detection
+        <https://arxiv.org/abs/2103.07461>`_.
+
+    Slightly different from original models:
+    the difference is:
+    Stage 1: Original CenterNet2 when calculating heatmap loss,
```

Comment:
Will this affect the performance

Reply:
@jshilong Centernet2 should be support faster-rcnn-retinanet, so instead of create a new centernet2head, create a new two-stage detector to give labels to one-stage head maybe a better practice?

Reply:
yeah, will look carefully at FCOS head and RetinaNet head closely to see whether we can build the head on top of them.

Reply:
@yyz561  Actually you can refactor you centernet head loss function to be same with fcos head, then create a new a new two stage, to give them same input. May  i write the new two stage in your branch?

Reply:
Sure, welcome to participate in this work!